sources/packages/backend/apps/db-migrations/src/db-migrations.service.ts

```diff
@@ -13,8 +34,10 @@ export class DBMigrationsService {
 
   /**
    * Run all pending migrations.
+   * @param failureBehavior the behavior to use when a failure occurs.
+   * Defaults to log only.
    */
-  async run(): Promise<void> {
+  async run(failureBehavior = FailureBehavior.LogOnly): Promise<void> {
```

Comment:
What is the benefit of setting the default value here?

Reply:
After further testing, I notice the RPEL is resilient to unhandled exceptions, hence I simplified the code, removing the `catch` and letting the exception be thrown all the times which would fix the migration issue for CI/CD and would also not affect the REPL mode with the benefit of simplified the code.